docs/core-concepts/dotnet-core.md

```diff
@@ -0,0 +1,67 @@
+
+The C#, F# and VB compilers all support targeting and running on .NET Core (the compilers are all managed). .NET Core tools are integrated into severals text editors and IDEs, including Visual Studio, Visual Studio Code, Sublime Text and Vim, providing .NET Core development experiences in your favorite environment. 
+
+.NET Core is open source and was contributed to the [.NET Foundation](http://dotnetfoundation.org) by Microsoft in 2014 and is now one of the most active .NET Foundation projects. It can be freely adopted by individuals and companies, including for personal, academic or commercial purposes. Multiple companies are using .NET Core as part of apps, tools, new platforms and hosting services. Some of these companies make significant contributions to .NET Core on GitHub and provide guidance on the product direction as part of the [.NET Foundation Technical Steering Group](http://www.dotnetfoundation.org/blog/tsg-welcome).
+
+.NET Core v1 supports the following workloads and app-models:
+
+- Class Libraries
+- Console apps
+- ASP.NET Core websites and services
+- UWP Windows 10 apps
+
+You can get started with .NET Core in just a few minutes:
+
+- [Getting started with .NET Core](https://dotnet.github.io)
+- [.NET Core samples](https://github.com/dotnet/core)
+- [.NET Core OS and CPU Support](link)
+- [Explore .NET Core repos](https://github.com/dotnet/core)
+
+Composition of .NET Core
+------------------------
+
+**.NET Core is the following:**
+
+- **A set of packages** -- These contain the runtime and the API shape and implementation of .NET Core.
+- **A set of metapackages (e.g. .NET Standard Library)** --  They describe various layers of .NET Core by referencing the appropriate set of library packages.
+- **A [distribution](https://dotnet.github.io) of .NET Core from Microsoft** -- This includes the CoreCLR runtime, associated libraries, the console application host and the `dotnet` launcher. It is described by the "Microsoft .NET Core Application" metapackage.
+- **A [distribution](https://dotnet.github.io) of the .NET Core SDK from Microsoft** -- This includes a set of tools for restoring NuGet packages and compiling and building apps, also controlled from the `dotnet` launcher. It is extensible, enabling anyone to add commands. The package also includes a copy of .NET Core to use with the tools. Typically, developers will install this distribution.
+
+**Closely affiliated with .NET Core:**
+
+- **The ".NET Standard Library"** - This is the API spec that describes the API evolution of .NET and that .NET Core implements. While many of the additions to the Standard Library will come from API additions to .NET Core, some will come from other platforms, such as the .NET Framework and Mono. There is a suble difference between the .NET Standard Library as a spec and as a meta-package. It's the distinction between the ref (reference assembly) and lib (implementation) folders in the .NET Standard Library packages.
```

Comment:
This doesn't explain that much to me. Again, this could be read by people who have no idea wtf ref assemblies are (they are not familiar with .NET). Maybe add a sentence explaining what each of the `ref` and `impl` does?